### ISSUE TITLE
[FR] Export to PDF

### ISSUE THREAD

**Issue by lariane-guerra** [AppFlowy-IO/AppFlowy#2938]
### Description

Its possible to export to pdf?

### Impact

Its will be good to share documentation with a team.
Like me, i take notes for test e2e e i cant send to my team.

**Comment by AdiAr11** [comment]
Hi @annieappflowy , I can try to implement this if this is in the current roadmap. 😃

**Comment by xiaodusx1** [comment]
yes,pdf is popular

**Comment by sandeeptiwari101** [comment]
Dear Developers, I have been using AppFlowy from last two days and have created beautiful notes using it all thanks to the wonderful software. But the very next thing I want to do is share my notes with others as pdf. However, there is no such functionality. This is a deal breaker. I found that a feature has been created in Git for this same issue but there is no one assigned to work on it since more than a year. **I'll humbly request to please prioritize it**. What is the use of using this software if you can't even export it to pdf and share it will multitude of people who just want to read your notes?

**Comment by fuchs-fabian** [comment]
Yes, it would be great to create a pdf or html document. It would also be really great to export everything at once.

**Comment by Tofoillote** [comment]
Hi there, you know I was thinking that it was so obvious that appflowy would export my notes in PDF that I didn't even check this before today, having a tons of notes and just wanted to share one.°

What a disapointment. 

Just I though this was about replacing notion, but it even can't "print" in a PDF printer.

Please take this as a priority into the roadmap.

Maybe going thru at least HTML document, that we can print from browsers.

**Comment by black-puppydog** [comment]
I am unsure what the implementation of https://github.com/AppFlowy-IO/appflowy-editor/pull/695 means for this ticket. is this still planned?

**Comment by henri9813** [comment]
Hello, any news on this ?

**Comment by rileyhawk1417** [comment]
There was a [PR](https://github.com/AppFlowy-IO/appflowy-editor/commit/f6ee2fc02c289414b44ff97d5209499a0e7ab601) that was merged into appflowy-editor. Since its not displayed in AppFlowy could be a WIP or something needs to be finalized before adding it in.

**Comment by annieappflowy** [comment]
Supported in v0.11.1
